scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -162,7 +165,7 @@ class FossId internal constructor(
         DELTA
     }
 
-    private val secretKeys = listOf("serverUrl", "apiKey", "user")
+    private val secretKeys = listOf("apiKey", "user")
```

Comment:
This change seems unrelated to the topic, strictly speaking, or?

Reply:
@sschuberth in my opinion since we add serverUrl to scan results, it doesn't make sense to hide it in scan options anymore. Please let me know if you disagree.

Reply:
Ok, makes sense.